geode-gfsh/src/main/java/org/apache/geode/management/internal/cli/functions/DataCommandFunction.java

```diff
@@ -314,17 +314,13 @@ public DataCommandResult remove(String key, String keyClass, String regionName,
         }
       } else {
         DataPolicy policy = region.getAttributes().getDataPolicy();
-        if (!policy.withPartitioning()) {
-          region.clear();
-          if (logger.isDebugEnabled()) {
-            logger.debug("Cleared all keys in the region - {}", regionName);
-          }
-          return DataCommandResult.createRemoveInfoResult(key, null, null,
-              CliStrings.format(CliStrings.REMOVE__MSG__CLEARED_ALL_CLEARS, regionName), true);
-        } else {
-          return DataCommandResult.createRemoveInfoResult(key, null, null,
-              CliStrings.REMOVE__MSG__CLEARALL_NOT_SUPPORTED_FOR_PARTITIONREGION, false);
+        region.clear();
```

Comment:
Can we add clear() instead of using remove()? Till the deprecated remove-all is removed, the remove-all can clear().

Reply:
I agree this could be confusing. I've added clear() as a helper method invoked by remove() when called with the 'ALL' flag to help make the relationship more visible.